src/features/offer/api/useSimilarOffers.ts

```diff
@@ -99,7 +101,8 @@ export const useSimilarOffers = ({
       }
     },
     {
-      enabled: !!categories && !!netInfo.isConnected,
+      staleTime: 1000 * 60 * 5,
+      enabled: !!categories && !!netInfo.isConnected && !!netInfo.isInternetReachable && isFocused,
```

Comment:
je pense qu'il serait intéressant d'en parler en synchro

je pense que `enabled: !!netInfo.isConnected && !!netInfo.isInternetReachable && isFocused,` serait bien partout par défaut sur toutes les requetes (avec parfois des trucs en plus comme là `!!categories`)

est-ce qu'il y aurait des contre-indications ?

Reply:
J'ai noté le point en synchro, je ne vois pas d'inconvénient à avoir `!!netInfo.isConnected && !!netInfo.isInternetReachable && isFocused` mais ça doit faire l'objet d'un autre ticket car ça va nécessiter de modifier pas mal de fichiers